package.json

```diff
@@ -36,6 +36,7 @@
         "vue-markdown": "^2.2.4",
```

Comment:
It would be nice if we can have list of shortcuts available somewhere, like in a modal. I like how most apps do it where you can press `shift` + `?` and it brings up a list of shortcuts.

Reply:
Good Idea but im not sure how i'd do this. (Not touched vue before, this is my first exposure) Just looking at your code and adding one and one together. I know how I'd open the model with a link, but if I attach the hotkey function to that button that opens the model, that would only work when the button is visible?

Reply:
No worries, looks awesome, great job!